tests/unit/moduleapi/commandfilter.tcl

```diff
@@ -141,3 +141,24 @@ test {Blocking Commands don't run through command filter when reprocessed} {
         assert_equal [r lpop list2{t}] 1
     }
 }
+
+test {Filtering based on client id} {
+    start_server {tags {"modules"}} {
+        r module load $testmodule log-key 0
+
+        set rd [redis_deferring_client]
```

Comment:
do we really need a deferring client here? or just a second client?
please also add `$rr close` at the end.
```suggestion
        set rr [redis_client]
```

Reply:
done.  adjusted throughout to simplify test